setup.py

```diff
@@ -2,21 +2,23 @@
 # -*- coding: utf-8 -*-
 import os
 
+# Get the version from the package
+from flex import VERSION
```

Comment:
That's the reason why `flex/__init__.py` was also updated to ensure that the import doesn't happen unless it has to be done.